lisa/tools/hyperv.py

```diff
@@ -381,30 +396,66 @@ def create_virtual_disk(self, name: str, pool_name: str, columns: int = 2) -> No
             force_run=True,
         )
 
-    def _check_exists(self) -> bool:
-        try:
-            self.node.tools[PowerShell].run_cmdlet(
-                "Get-VM",
-                force_run=True,
-            )
-            self._log.debug("Hyper-V is installed")
-            return True
-        except LisaException as e:
-            self._log.debug(f"Hyper-V not installed: {e}")
-            return False
+    # This method is specifically for Azure Windows Server edition VMs
+    # with internal NAT switch.
+    # Do not use this method in Hyper-V hosts on LAN or other networks where DHCP
+    # server is already available.
+    # This method will configure the DHCP server on the Hyper-V host to provide IP
+    # addresses to VMs.
+    # Reference doc:
+    # https://techcommunity.microsoft.com/blog/itopstalkblog/how-to-setup-nested-virtualization-for-azure-vmvhd/1115338  # noqa
+    def configure_dhcp(self, dhcp_scope_name: str = "DHCPInternalNAT") -> None:
```

Comment:
I don't see this method is called in this PR, how will it be called?

Reply:
It will be used in "HyperVPreparationTransformer" transformer. I will raise the PR after this one.